bin/ham-pm2

```diff
@@ -35,20 +35,20 @@ usage() {
     echo ""
     echo "    source                           To be sourced at the top of a script to warm up the pm2"
     echo "                                     environment to run it multiple times."
-    echo "    start-once APP_NAME START_CMD    Start a pm2 process if it isnt already started."
+    echo "    start-once PM2_APP_NAME START_CMD    Start a pm2 process if it isnt already started."
```

Comment:
We dont need to rename this here since its for documentation. That being said as long as the columns are aligned it's fine.

As it is it breaks the alignment with the source description.

Reply:
Fixed the alignment in commit 02fbe99 before merging